### PR TITLE
Add new version of Datadog package.

### PR DESCRIPTION
**Pull request by hkaj**

### CODE REVIEW

File: repo/packages/D/datadog/1/marathon.json.mustache [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            ]
+        },
+        "volumes": [
+            {
+                "containerPath": "/var/run/docker.sock",
+                "hostPath": "/var/run/docker.sock",
+                "mode": "RW"
+            },
+            {
+                "containerPath": "/host/proc/mounts",
```
Comment (remh) [inline repo/packages/D/datadog/1/marathon.json.mustache]: should be `/host/proc`

File: repo/packages/D/datadog/1/marathon.json.mustache [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            }
+        ]
+    },
+    "cpus": {{datadog.cpus}},
+    "mem": {{datadog.mem}},
+    "instances": {{datadog.instances}},
+    "acceptedResourceRoles": ["slave_public", "*"],
+    "constraints": [
+        ["hostname", "UNIQUE"],
+	["hostname", "GROUP_BY"]
```
Comment (remh) [inline repo/packages/D/datadog/1/marathon.json.mustache]: nitpick, fix alignment

File: repo/packages/D/datadog/1/resource.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "???????????????": "datadog/docker-dd-agent:11.2.583"
```
Comment (remh) [inline repo/packages/D/datadog/1/resource.json]: is that normal ?

Comment (hkaj) [inline repo/packages/D/datadog/1/resource.json]: This should be the image ID to use. Once we have a build for the docker image I'll update it and PR against mesosphere/universe.

Comment (remh) [inline repo/packages/D/datadog/1/resource.json]: 👍

File: repo/packages/D/datadog/1/marathon.json.mustache [hunk 4]
```diff
@@ -0,0 +1,62 @@
+{
+    "id": "{{datadog.app_id}}",
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "{{resource.assets.container.docker.?????????????}}",
+            "parameters": [
```
Comment (hkaj) [inline repo/packages/D/datadog/1/marathon.json.mustache]: as we discussed I added `MESOS_SLAVE` and we'll advise running a container manually on the master with `MESOS_MASTER` to enable the mesos_master check and also collect system metrics from the master host.

**Pull request closed without merging**